airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -351,6 +353,8 @@ def __init__(
         s3_prefix: str = "",
         export_only: list[str] | None = None,
         wait_for_completion: bool = True,
+        waiter_interval: int = 30,
+        waiter_max_attempts: int = 40,
```

Comment:
20 minutes by default feels like a long time. Is that reasonable for this context?

Reply:
That's the default waiting time defined in the hook wait_for_export_task_state. I've added these settings as in us-east-1 we're getting a 20 minutes delay before the actual export task starts, resulting in the failure of the DAG, so I need a way to modify those values and increase them over 30 minutes.

Reply:
Agree, we have to set these values to be backward compatible

Reply:
Cool, fair enough, lgtm!